UI/QuickMenu/IButtonPage.cs

```diff
@@ -8,6 +8,8 @@ public interface IButtonPage
         ReMenuButton AddButton(string text, string tooltip, Action onClick, Sprite sprite = null);
         ReMenuToggle AddToggle(string text, string tooltip, Action<bool> onToggle, bool defaultValue = false);
         ReMenuToggle AddToggle(string text, string tooltip, ConfigValue<bool> configValue);
+        ReMenuSlider AddSlider(string text, string tooltip, Action<float> onSlide, float defaultValue = 0, float minValue = 0, float maxValue = 10);
```

Comment:
Should this be part of IButtonPage? Should the SliderCategory be a button page?
Considering it should be its own isolated page that should only allow sliders, I don't know if that makes sense.

What does it look like if you were to add a button to a slider category? Or a slider to a normal category.